src/Core/Settings.h

```diff
@@ -432,7 +433,7 @@ class IColumn;
     M(Bool, allow_experimental_map_type, false, "Allow data type Map", 0) \
     M(Bool, allow_experimental_window_functions, false, "Allow experimental window functions", 0) \
     M(Bool, use_antlr_parser, false, "Parse incoming queries using ANTLR-generated experimental parser", 0) \
-    M(Bool, async_socket_for_remote, false, "Asynchronously read from socket executing remote query", 0) \
+    M(Bool, async_socket_for_remote, true, "Asynchronously read from socket executing remote query", 0) \
```

Comment:
@KochetovNicolai maybe it worth enabling this setting by default in a separate PR? (so that it can be reverted separately and to underline it in changelog)

Reply:
Ok, maybe ...